astroquery/vizier/core.py

```diff
@@ -370,22 +373,35 @@ def query_region_async(self, coordinates, radius=None, inner_radius=None,
 
         # Process coordinates
         if isinstance(coordinates, (commons.CoordClasses,) + six.string_types):
-            c = commons.parse_coordinates(coordinates).transform_to('fk5')
+            c = commons.parse_coordinates(coordinates).transform_to(frame)
 
             if not c.isscalar:
                 center["-c"] = []
                 for pos in c:
-                    ra_deg = pos.ra.to_string(unit="deg", decimal=True,
-                                              precision=8)
-                    dec_deg = pos.dec.to_string(unit="deg", decimal=True,
+                    if frame == 'galactic':
+                        glon_deg = pos.l.to_string(unit="deg", decimal=True,
+                                                precision=8)
```

Comment:
Fix indent, or keep this in one line. Latter would be more readable, and we don't enforce limit of 80 line length, it's perfectly fine to let them go a bit longer (e.g. 100). 

Do the same for lines below, too.